frontend/src/components/account/Register.jsx

```diff
@@ -142,26 +155,80 @@ export default function Register() {
     catch (e) {
       return;
     }
-    //go to the next step   
-    setActiveStep(activeStep + 1);
+
+    // Go to the confirmation page
+    setActiveStep(2);
   };
 
   const account = (
-    <Grid container direction={"column"} spacing={2}>
-      <Grid item>
-        <TextField required id="standard-basic" variant="outlined" label="Email Address" onChange={e => setUsername(e.target.value)} />
-      </Grid>
-
-      <Grid item>
-        <TextField type={'password'} required id="standard-basic" variant="outlined" label="Password" onChange={e => setPassword(e.target.value)} />
+    <Grid
```

Comment:
Might improve readability/understandability if we make some of these into their own components then just imported them. Especially since you already do this for PhoneNumber

Reply:
We won't make this change right now, but it is definitely something to consider in the future!